api/descriptions/feeds/custom_vulnerabilities_get.md

```diff
@@ -1,17 +1,41 @@
-Retrieves the list of vulnerabilities for internally developed packages.
+Retrieves the list of custom vulnerabilities and associated rules for handling internally created or packaged apps.
 
-These entries are used by the Twistlock scanner to detect vulnerable custom components (apps, libraries, etc) that were developed and packaged internally.
+This list is used by the Twistlock scanner to detect vulnerable custom components (apps, libraries, etc) that were developed and packaged internally.
 
-**NOTE**: When a vulnerable custom component is detected in an image, you must have a rule to tell Twistlock how to handle it.
-Vulnerability rules can be created in the Console UI or with the /api/v1/policies/cve endpoint.
-Create a vulnerability rule and set ID 412 (Image contains vulnerable custom components) to ignore, alert, or block.
+**Note:** When a vulnerable custom component is detected in an image, you must have a rule to tell Twistlock how to handle it. Vulnerability rules can be created using the Console UI or with the /api/v1/policies/cve endpoint. To set a vulnerability rule to ignore, alert, or block, use the rule ID `412` to indicate the image contains vulnerable custom components. See `/api/v1/policies/cve` endpoint for more info.
 
-The following example curl command retrieves the list of custom-defined vulnerabilities:
+### cURL Request
+
+The following cURL command retrieves a list of all the custom vulnerabilities and associated rules.
+
+**Note:** The cURL will require you to enter the user's password after submission.
```

Comment:
@KICKdesigns Not sure if we need this note. Maybe we just have a section at the top of the API docs that describe all the ways you can auth with an endpoint. Example:

* Get a token, pass the token in the request header.
* Call curl with the username only, curl prompts for a password. More secure, because your terminal history won't contain the password.
* Call curl with the -p option and the password so that there's no interactive second step
* etc

Reply:
@iansk Will put this in the getting started section of the docs and remove the rest of the references.